runtime/port/module.xml

```diff
@@ -107,6 +107,7 @@
 			<include-if condition="spec.zos_390.*"/>
 			<include-if condition="spec.linux_ztpf.*"/>
 			<exclude-if condition="spec.linux_x86.*"/>
+			<exclude-if condition="spec.osx_x86-64"/>
```

Comment:
None of the `include-if` elements match on osx so this should not be necessary.

Reply:
That should be true.  I think I probably saw the exclude for linux and figured to add for osx considering the name, but the case should have been to remove this line.  Agreed?

Reply:
There are several redundant `<exclude-if>` elements in this file: I'll let you decide whether you want to remove them all, but let's not add more.

Reply:
I just removed the add.  Cleanup can be performed under another issue.